### ISSUE TITLE
xfce terminal support for solarized is baked into xfce4-terminal now; it should be removed?

### ISSUE THREAD

**Issue by nfirvine** [altercation/solarized#322]
http://git.xfce.org/apps/xfce4-terminal/commit/colorschemes/solarized-light.theme.in?id=0aab08157c72b4b76587cf89f8d4f1f235f2da24

Especially considering upstream is unmaintained: https://github.com/sgerrand/xfce4-terminal-colors-solarized/issues/11

**Comment by nfirvine** [comment]
As per https://hashicorp.com/blog/atlas-general-availability.html , turns out it is maintained.

**Comment by TrevorBramble** [comment]
@nfirvine Is that the intended URL? I don't see anything relevant. We should probably keep the port around for people who run earlier XFCE versions...

**Comment by nfirvine** [comment]
@TrevorBramble No, it wasn't; updated. May it would be useful to add a notes somewhere that mentions that it's baked into new XFCEs and is also incompatible. Not sure where that's appropriate.

**Comment by egmontkob** [comment]
Let's also note that `xfce4-terminal/light/terminalrc` is broken: ColorPalette5's green channel is incorrect, plus entries 11-12 and 13-15 are swapped. One more reason to retire it here.
